config/samples/kustomization.yaml

```diff
@@ -1,3 +1,3 @@
 resources:
-  - classicFullStack.yaml
+  - cloudNativeFullStack.yaml
```

Comment:
that is unfortunatly not possible, since cloudNative is not running on OLM, because it needs the CSI Driver, which can not be applied because we create in through a manifest and OLM does not support daemonsets in manifests

Reply:
Ammended/Removed this change (now defaults to `applicationMonitoring.yaml` as intended)